lib/src/days_view.dart

```diff
@@ -1,8 +1,7 @@
 part of clean_nepali_calendar;
 
 typedef HeaderDayBuilder = Widget Function(String headerName, int dayNumber);
-
-const double _kDayPickerRowHeight = 40.0;
+const double _kDayPickerRowHeight = 52.0;
```

Comment:
May be we should different value for when both dates are visible and when only nepali date is visible?